client/Lykke.Service.Subscribers.Client/Lykke.Service.Subscribers.Client.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Version>1.0.1</Version>
+    <Authors>Lykke</Authors>
+    <Company>Lykke</Company>
+    <Product>Lykke.Service.Subscribers</Product>
+    <Description>Client for Lykke.Service.Subscribers API</Description>
+    <PackageLicenseUrl>https://github.com/LykkeCity/Lykke.Service.Subscribers/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/LykkeCity/Lykke.Service.Subscribers</PackageProjectUrl>
+    <PackageIconUrl>https://avatars3.githubusercontent.com/u/14153330?v=3&amp;s=200</PackageIconUrl>
+    <RepositoryUrl>https://github.com/LykkeCity/Lykke.Service.Subscribers.git</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+    <PackageTags>Lykke</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="4.2.0" />
+    <PackageReference Include="Lykke.Common" Version="3.0.0" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="2.3.10" />
+  </ItemGroup>
+
```

Comment:
Include XML documentation for debug and release

Reply:
@MikhailGerasimov  What kind of documentation? Give me an example where do you place this documentation?